### ISSUE TITLE
Is this still in use?

### ISSUE THREAD

**Issue by boffbowsh** [alphagov/search-performance-dashboard#5]
@MatMoore does your team use this or can it be moved to the attic?

**Comment by MatMoore** [comment]
Nope, we're using data studio for this kind of thing now so I think we can move this to the attic.
